### PR TITLE
Hello Microverse Homepage

### PR DESCRIPTION
**Pull request by NayanKaran**
Added Homepage showing "Hello Microverse" and added styles to it along with project details and project license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+- Open the index.html file.
+
+
+
+## Authors
+
+👤 **Debabrata Mandal (Nayan Karan)**
+
+- GitHub: [@githubhandle](https://github.com/NayanKaran)
+
```
Comment (vic778) [inline README.md:36]: - Optional [] kindly add your LinkedIn account here or twitter

**Review by vic778 — APPROVED** [review]
Hi @NayanKaran   , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Charlie Brown

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (NayanKaran) [reply README.md:36]: @vic778 can I merge the project now?

Comment (vic778) [reply README.md:36]: @NayanKaran yes you can marge it.

**Comment by NayanKaran** [comment]
@vic778 can I merge the project now?

File: README.md (hunk 1 above)
Comment (NayanKaran) [reply README.md:36]: Thank you.

**Pull request merged**